libbeat/common/kafka/version.go

```diff
@@ -17,7 +17,16 @@
 
 package kafka
 
-import "github.com/Shopify/sarama"
+import (
+	"fmt"
+
+	"github.com/Shopify/sarama"
+)
+
+// Version is a kafka version
+type Version struct {
+	String string
+}
```

Comment:
We could have the same features of the validation by using `type Version string` without having to have a String Field.

Reply:
Agree, changed